src/app/features/admin/components/carriage-preview/carriage-preview.component.ts

```diff
@@ -1,25 +1,42 @@
-import { NgFor } from '@angular/common';
-import { ChangeDetectionStrategy, Component, effect, input, signal } from '@angular/core';
+import { NgClass, NgFor } from '@angular/common';
+import { ChangeDetectionStrategy, Component, effect, input, output, signal } from '@angular/core';
 import { Carriage } from '@features/admin/interfaces/carriage.interface';
+import { BookSeats } from '@features/search/services/trip/trip.service';
 import { TuiAppearance, TuiSurface, TuiTitle } from '@taiga-ui/core';
 import { TuiCardLarge, TuiHeader } from '@taiga-ui/layout';
 
 @Component({
   selector: 'dd-carriage-preview',
   standalone: true,
-  imports: [NgFor, TuiCardLarge, TuiSurface, TuiTitle, TuiHeader, TuiAppearance],
+  imports: [NgFor, TuiCardLarge, TuiSurface, TuiTitle, TuiHeader, TuiAppearance, NgClass],
   templateUrl: './carriage-preview.component.html',
   styleUrl: './carriage-preview.component.scss',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CarriagePreviewComponent {
+  public options = input<{ isClick: boolean; isShowTitle: boolean }>({
```

Comment:
{ isClick: boolean; isShowTitle: boolean } - seems like interface

Reply:
This is just an object so as not to make 2 separate input signals. these variables are not particularly related to each other. is there a need for an interface?

Reply:
Ok, if it seems useless then leave it just like that👌🏻